app/src/test/java/com/github/se/orator/model/symblAi/AudioRecorderTest.kt

```diff
@@ -100,7 +100,7 @@ class AudioRecorderTest {
           }
           .thenReturn(PackageManager.PERMISSION_DENIED)
 
-      audioRecorder.startRecording()
+      audioRecorder.startRecording(File(context.cacheDir, "audio_record.wav"))
     }
   }
 
```

Comment:
You could also add a test to verify that providing a custom File object to startRecording(audioFile: File) indeed results in the recording being saved under that specific file name. However, this is not mandatory for now and could be done in next sprint.

Reply:
Good idea, I'll implement this in the next commit.